cmd/login/clientcert.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/giantswarm/backoff"
 	"github.com/giantswarm/microerror"
 	"github.com/spf13/afero"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Sorry, I missed this before. Can we use `sigs.k8s.io/yaml`? It's mentioned in [fmt](https://github.com/giantswarm/fmt/blob/master/go/yaml_marshalling.md)

Reply:
Yeah I also just noticed that. I copied this function from gsctl so the older dependencies came along.